crates/viewer/re_viewer/src/ui/recordings_panel.rs

```diff
@@ -114,7 +114,12 @@ fn recording_list_ui(
 
     servers.server_list_ui(ui, ctx, remote_recordings);
 
-    if local_recordings.is_empty() && welcome_screen_state.hide {
+    // Show placeholder message if there's absolutely nothing else to show.
+    if ctx.storage_context.tables.is_empty()
+        && servers.is_empty()
```

Comment:
note that due to recent changes presence of a remote recording implies presence of a server